tests/cypress/e2e/jcontent/breadcrumbsPageBuilder.cy.ts

```diff
@@ -10,25 +10,20 @@ describe('Breadcrumbs inside boxes of page builder', () => {
         const pageBuilder = new JContentPageBuilder(jcontent);
         const module = pageBuilder.getModule('/sites/digitall/home/landing/slider/innovating-technologies', false);
         module.click();
-        module.getHeader().select();
         const breadcrumbs = module.getFooter().getItemPathDropdown().open();
         breadcrumbs.shouldHaveCount(2);
-
         breadcrumbs.select('landing');
-
-        jcontent.getSelectionDropdown().get().find('span').should('have.text', '1 item selected');
+        pageBuilder.getModule('/sites/digitall/home/landing/slider/innovating-technologies').hasNoHeaderAndFooter();
```

Comment:
Looks exactly the same as the bottom test. Remove 'Selects multiple elements' test and maybe update the description to something like 'Verifies breadcrumbs are displayed and can be used to click on element (not selected)'

Reply:
can we also update description? It looks like test is not doing any selection anymore.